src/ZPublisher/Converters.py

```diff
@@ -22,14 +23,19 @@
 default_encoding = 'utf-8'
 
 
-def field2string(v):
-    """Converts value to string."""
+def convert_single_value_to_string(v):
```

Comment:
I'm confused, why did you create this additional function? Its body can go into `field2string` and then you can `field2string` where needed instead of this extra method.

Reply:
Meanwhile the code for the module is way easier to read - this was not the case a couple of months ago.

The `field2xxx` converters called each other, partly multiple times, and the called `field2xxx` converter called another `field2xxx` converter. It was very hard to reason about the code.

I extracted the `value_to_string` into a separate function to clearly separate between what is a field converter and what is a common base function.

It is up to you - I could prefix `convert_single_value_to_string` with an underscore to show my intention, or I revert the change and let other converters call `field2string` again.

Reply:
My vote is for reverting that part of the change and let other converters just call `field2string`. I always found it logical and easy to follow that converters for compound values such as lines or tokens would utilize the converters for simple values internally.

Reply:
Very well!